tests/test_fixtures.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA-pytest-Commmons test fixtures."""
```

Comment:
Pytest-REANA now :wink:

Reply:
Done. I have used lowercase `pytest` since it is the [`default in cookiecutter-pytest-plugin`](https://github.com/pytest-dev/cookiecutter-pytest-plugin/blob/master/pytest-%7B%7Bcookiecutter.plugin_name%7D%7D/setup.py#L15).